scripts/build_imagemagick

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+set -e
+
+WORK_DIR=/usr/src
+
+function handle_error() {
+  local status=$?
+  if [[ $status > 0 ]]; then
+    echo "🔥 There was a problem building the binary. Operation aborted."
+    exit 1
+  fi
+}
+trap handle_error EXIT SIGINT
+
+function add_xenial_repositories() {
+  cat <<-EOT >> /etc/apt/sources.list
+  deb http://archive.ubuntu.com/ubuntu/ xenial main universe
+  deb http://security.ubuntu.com/ubuntu xenial-security main
+EOT
+}
+
+# add source packages repository to sources.list
+function add_source_repositories() {
+  echo "deb-src http://archive.ubuntu.com/ubuntu/ xenial main universe" >> /etc/apt/sources.list
+  if [[ $STACK == "heroku-20" ]]; then
+    add_xenial_repositories
+  fi
+}
+
+add_source_repositories
+
+echo "⚙️ Getting libraries and dependencies..."
+apt-get update -y
+apt-get install -y build-essential autoconf libtool git-core
```

Comment:
From here, most of the steps are extracted from the existing readme file, with a few tweaks.